File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { WaifuFinderSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await WaifuFinderSDK.test()
    equal(null !== testsdk, true)
  })

})

File: rb/test/exists_test.rb
# WaifuFinder SDK exists test

require "minitest/autorun"
require_relative "../WaifuFinder_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = WaifuFinderSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
